hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/OzoneClientAdapter.java

```diff
@@ -95,7 +97,9 @@ SnapshotDiffReport getSnapshotDiffReport(Path snapshotDir,
       String fromSnapshot, String toSnapshot)
       throws IOException, InterruptedException;
 
-  boolean recoverLease(String pathStr) throws IOException;
+  List<OmKeyInfo> recoverFilePrepare(String pathStr) throws IOException;
```

Comment:
openKeyInfo has all the allocated blocks. If the file has multiple preallocated blocks, we actually don't know how many blocks are actually used, because each block in openKeyInfo only has the allocated block length, doesn't have the real data length. And keyInfo will have real data length for each block, but keyInfo will not have preallocated by not used yet blocks info.